### DISCUSSION TITLE
Reset password and cant login

### DISCUSSION THREAD

**Discussion by nezaboravi** [npm/feedback#926] [Website]
I have filed a support request Ticket ID: 2144253
I cant login into website. 
I used my saved password in password manager, and website responded with wrong username and password.
I clean cookies, clicked to reset password, got email, followed the link, changed password and tried to login again. 
Same message - wrong username or password. 
Then i tried to use newly created password in another browser, got same message.  
What to do?

**Comment by wongchichong** [comment]
I use the username as username, not email as username,
problem solved.

**Comment by monishcm** [comment]
Looks like the support issue is already addressed.
